### PR TITLE
Improve loading state in activity log

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5870

https://user-images.githubusercontent.com/12435965/185213050-c7ce7e23-1b6b-49c2-8758-d5fe44550fcc.mp4

### CODE REVIEW

File: components/admin-panel/sections/ActivityLog/index.js [hunk 1]
```diff
@@ -88,12 +89,18 @@ const ActivityLog = ({ accountSlug }) => {
     });
   };
 
+  if (error) {
+    return <MessageBoxGraphqlError error={error} />;
+  }
+
```
Comment (Betree) [inline components/admin-panel/sections/ActivityLog/index.js:95]: We should not block the UI if there's an error, people should be able to change their filters

**Review by Betree — COMMENTED** [review]
Looking good, linting issuee to fix

File: components/admin-panel/sections/ActivityLog/index.js (hunk 1 above)
Comment (SudharakaP) [reply components/admin-panel/sections/ActivityLog/index.js]: This is a very simple fix I felt so I've move the error within the filters. Now the error should appear along with the filters 👍🏽

**Comment by SudharakaP** [comment]
> Looking good, linting issue to fix

Done. 👍🏽

**Review by Betree — APPROVED** [review]

**Pull request merged**